index.js

```diff
@@ -1 +1,28 @@
-// Add your code here
+function submitData(name, email) {
+  const userData = {
+    name: `${name}`,
+    email: `${email}`,
+  }
```

Comment:
Two ways to simplify this:

```suggestion
  const userData = {
    name: name,
    email: email,
  }
```
OR some good ol JS destructuring 
```suggestion
  const userData = {
    name,
    email
  }
```